docs/content/docs/ci/CIGradlePlugin.md

```diff
@@ -38,57 +38,143 @@ builds {
+}
 ```
 
-- `performanceTests` - запустить перформанс тесты модуля
+#### Performance tests
+
+Runs performance tests.
 
 ```groovy
-    fastCheck {
-        performanceTests {
-          configuration = "configuration name"
-          enabled = true // true by default
-        }
-    }
+performanceTests {
+  configuration = "configuration name" // performance configuration from Instrumentation plugin
+  enabled = true // true by default
+}
+```
+
+#### Android lint
+
+Run [Android lint]({{< ref "/docs/checks/AndroidLint.md" >}}) tasks.
+
+```groovy
+lint {}
+```
+
+#### Compile UI tests
+
+Compile instrumentation tests. It is helpful in local development.
+
+```groovy
+compileUiTests {}
+```
+
+#### Unit tests
+
+Run unit tests.
+
+```groovy
+unitTests {}
+```
+
+#### Upload to QApps
+
+{{< hint info>}} This section contains Avito specific information {{< /hint >}}
+
+Upload [artifacts]({{< relref "#collecting-artifacts">}}) to QApps (internal system )
+
+```groovy
+artifacts {
+    apk("debugApk", ...)
+}
+uploadToQapps {
+    artifacts = ["debugApk"]
+}
+```
+
+#### Upload to Artifactory
+
+Upload [artifacts]({{< relref "#collecting-artifacts">}}) to Artifactory.
+
+```groovy
+artifacts {
+    file("myReport", "${project.buildDir}/reports/my_report.json")
+}
+uploadToArtifactory {
+    artifacts = ["myReport"]
+}
 ```
 
-enabled - запускать ли шаг. \
-configuration - имя performance конфигурации, которая объявленна в instrumentation plugin
+#### Upload to Prosector
 
-- `compileUiTests` - скомпилировать androidTest модуля
-- `unitTests` - запустить юнит тесты модуля и всех его зависимостей
-- `lint` - запустить android lint по настройкам в модуле
-- `docsDeploy` - опубликовать документацию в k8s (отключено)
-- `docsCheck` - запустить проверки документации
-- `uploadToQapps` - загрузить указанные артефакты в QApps
-- `uploadToArtifactory` - загрузить указанные артефакты в Artifactory
-- `uploadToProsector` - загрузить указанные артефакты в [Prosector (internal)](http://links.k.avito.ru/cfxrREPBQ)
-- `uploadBuildResult` - загрузить указанные артефакты в сервис релизов
-- `deploy` - загрузить указанные артефакты в google play
-- `artifacts` - зарегистрировать артефакты в качестве результатов билда.\
-Добавит проверку на их наличие и возможность загрузить при помощи других тасок.
+{{< hint info>}} This section contains Avito specific information {{< /hint >}}
+
+Upload [artifacts]({{< relref "#collecting-artifacts">}}) to [Prosector (internal)](http://links.k.avito.ru/cfxrREPBQ).
+
+```groovy
+artifacts {
+    apk("debugApk", ...)
+}
+uploadToProsector {
+    artifacts = ["debugApk"]
+}
+```
+
+#### Upload build results
+
+{{< hint info>}} This section contains Avito specific information {{< /hint >}}
+
+Upload all build results to a deploy service.
+
+```groovy
+uploadBuildResult {
+    uiTestConfiguration = "regression" // instrumentation configuration
+}
+```
+
+#### Deploy to Google Play
+
+Deploy to Google play.
```

Comment:
Deploy works with the config. Mark step internal

Reply:
fixed